surveys/2024-annual-survey/translations/de.md

```diff
@@ -66,18 +66,18 @@ Type: select one
 Type: select one
 
 - Ich versuche immer noch *aktiv*, Rust zu lernen
-- Im Jahr 2023
-- Im Jahr 2021 oder 2022
-- Im Jahr 2019 oder 2020
-- Im Jahr 2017 oder 2018
-- Im Jahr 2016 oder früher
+- Im Jahr 2024
+- Im Jahr 2022 oder 2023
+- Im Jahr 2020 oder 2021
+- Im Jahr 2018 oder 2019
+- Im Jahr 2017 oder früher
```

Comment:
Looks like German was the only language where this had not been adjusted; hopefully it does not skew the results

Reply:
this is unfortunate